src/System.Memory/src/System.Memory.csproj

```diff
@@ -0,0 +1,77 @@
+    <ILLinkClearInitLocals>true</ILLinkClearInitLocals>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) != '.NETFramework'">
+    <EnableDefaultItems>false</EnableDefaultItems>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="$(CommonPath)\CoreLib\System\Buffers\Text\FormattingHelpers.CountDigits.cs" Link="Common\CoreLib\System\Buffers\Text\FormattingHelpers.CountDigits.cs" />
+    <Compile Include="$(CommonPath)\System\MutableDecimal.cs" Link="Common\System\MutableDecimal.cs" />
+    <Compile Include="$(CommonPath)\System\Numerics\Hashing\HashHelpers.cs" Link="Common\System\Numerics\Hashing\HashHelpers.cs" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(IsPartialFacadeAssembly)' != 'true'">
+    <Compile Include="$(CommonPath)\CoreLib\System\Buffers\IMemoryOwner.cs" Link="Common\CoreLib\System\Buffers\IMemoryOwner.cs" />
+    <Compile Include="$(CommonPath)\CoreLib\System\Buffers\IPinnable.cs" Link="Common\CoreLib\System\Buffers\IPinnable.cs" />
+    <Compile Include="$(CommonPath)\CoreLib\System\Buffers\MemoryHandle.cs" Link="Common\CoreLib\System\Buffers\MemoryHandle.cs" />
+    <Compile Include="$(CommonPath)\CoreLib\System\Buffers\MemoryManager.cs" Link="Common\CoreLib\System\Buffers\MemoryManager.cs" />
+    <Compile Include="$(CommonPath)\CoreLib\System\Memory.cs" Link="Common\CoreLib\System\Memory.cs" />
+    <Compile Include="$(CommonPath)\CoreLib\System\MemoryDebugView.cs" Link="Common\CoreLib\System\MemoryDebugView.cs" />
+    <Compile Include="$(CommonPath)\CoreLib\System\MemoryExtensions.cs" Link="Common\CoreLib\System\MemoryExtensions.cs" />
+    <Compile Include="$(CommonPath)\CoreLib\System\ReadOnlyMemory.cs" Link="Common\CoreLib\System\ReadOnlyMemory.cs" />
+    <Compile Include="$(CommonPath)\CoreLib\System\ReadOnlySpan.cs" Link="Common\CoreLib\System\ReadOnlySpan.cs" />
+    <Compile Include="$(CommonPath)\CoreLib\System\Runtime\InteropServices\MemoryMarshal.cs" Link="Common\CoreLib\System\Runtime\InteropServices\MemoryMarshal.cs" />
+    <Compile Include="$(CommonPath)\CoreLib\System\Span.cs" Link="Common\CoreLib\System\Span.cs" />
+    <Compile Include="$(CommonPath)\CoreLib\System\SpanDebugView.cs" Link="Common\CoreLib\System\SpanDebugView.cs" />
+    <Compile Include="$(CommonPath)\CoreLib\System\SpanHelpers.BinarySearch.cs" Link="Common\CoreLib\System\SpanHelpers.BinarySearch.cs" />
+    <Compile Include="$(CommonPath)\CoreLib\System\SpanHelpers.Byte.cs" Link="Common\CoreLib\System\SpanHelpers.Byte.cs" />
+    <Compile Include="$(CommonPath)\CoreLib\System\SpanHelpers.Char.cs" Link="Common\CoreLib\System\SpanHelpers.Char.cs" />
+    <Compile Include="$(CommonPath)\CoreLib\System\SpanHelpers.T.cs" Link="Common\CoreLib\System\SpanHelpers.T.cs" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(IsPartialFacadeAssembly)' != 'true' and $([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) != '.NETFramework'">
```

Comment:
Interesting. Why don't we include these sources when targeting .NET Framework?

Reply:
And also by investigating at the build errors I was seeing, along with my interpretation from reading the old 2.1 csproj and the binlog: [/src/System.Memory/src/System.Memory.csproj](https://github.com/dotnet/corefx/blob/8b6f40185c25372943ac4a8ab94b860008ad2120/src/System.Memory/src/System.Memory.csproj)

Do you think it's incorrect?

Reply:
I'd be very surprised if we exclude public API like `ReadOnlySequenceSegment` on .NETFramework.

I don't see any NETFramework conditions on compile items in the original System.Memory.csproj.

Reply:
Somehow the first portion of my comment above got truncated when I posted it.

I fixed the problems with the ItemGroup conditions, PTAL at my latest commits.